cmake-scripts/AutoInstaller/CMakeLists.txt

```diff
@@ -1,5 +1,19 @@
-cmake_minimum_required(VERSION 2.8.8)
+cmake_minimum_required(VERSION 2.8.12)
+
+# Available CMake options
+
+# GCC_INSTALL_PREFIX
+#   This flag denote the gcc installation to let Clang++ uses stdc++ from it.
+#
+# CMAKE_PREFIX_PATH
+#   This flag denote the place to search tools, like python.
+#
+# LLVM_RECOMMEND_VERSION
+#   The LLVM verison wantted, from 3.1 to 3.6.2
```

Comment:
Currently, LLVM-Tracer is not compatible with LLVM 3.6.

LLVM seems make a great change on metadata data structure since version 3.6. Originally, llvm::MDNode is inherit from llvm::value. Since version 3.6, llvm::MDNode is inherit from a new class llvm::metadata, not depend on llvm::value anymore.

In SlotTracer.h of LLVM-Tracer, it uses llvm::MDNode::isFunctionLocal(), which is missing on version 3.6. I don't find an easy solution on it.